test/src/test/java/io/varhttp/FilterOrderTest.java

```diff
@@ -0,0 +1,92 @@
+package io.varhttp;
+
+import io.odinjector.OdinJector;
+import io.varhttp.filterorder.DefaultFilter1;
+import io.varhttp.filterorder.DefaultFilter2;
+import io.varhttp.filterorder.DefaultFilter3;
+import io.varhttp.filterorder.DefaultFilter4;
+import io.varhttp.controllers.withfilters.FilterCatcher;
+import io.varhttp.filterorder.FilterControllerClass;
+import io.varhttp.filterorder.ShouldNotBeRunFilterInner;
+import io.varhttp.filterorder.ShouldNotBeRunFilterOnTheSide;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class FilterOrderTest {
+	static Standalone launcher;
+	static Thread thread;
+	private static FilterCatcher filterCatcher;
+
+	@BeforeClass
+	public static void setup() {
+		OdinJector odinJector = OdinJector.create().addContext(new OdinContext(new VarConfig().setPort(8088)));
+		filterCatcher = odinJector.getInstance(FilterCatcher.class);
+		launcher = odinJector.getInstance(Standalone.class);
+		launcher.configure(config -> {
+			config.addDefaultVarFilter(DefaultFilter1.class);
+			config.addDefaultVarFilter(DefaultFilter2.class);
+			config.configure(configInner -> {
+				configInner.configure(configFurtherIn -> {
+					// This filter should not be run for FilterControllerClass as it is in an inner configurataion
+					configFurtherIn.addDefaultVarFilter(ShouldNotBeRunFilterInner.class);
+				});
+				config.addDefaultVarFilter(DefaultFilter3.class);
```

Comment:
it seems that this is using the `config` instead of `configInner` and thats why the tests are passing 😬

Reply:
should be fine now :-)